samcli/local/apigw/route.py

```diff
@@ -0,0 +1,85 @@
+"""
+Route definition for local start-api
+"""
```

Comment:
Moved the `Route` class definition since it didn't need to be in the `local_apigw_service.py` file and this helps with refactoring the file.